core/scss/components/brand-bar/_brand-bar.scss

```diff
@@ -0,0 +1,36 @@
+@charset "UTF-8";
+
+//
+// Brand Bar
+//
+// Stanford brand bar with the wordmark logo that links to the Stanford homepage. The brand bar sits at the very top of a website.
+//
+// .su-brand-bar - Default brand bar with white wordmark on cardinal red background
+// .su-brand-bar--bright - White wordmark on bright red background
+// .su-brand-bar--dark - White wordmark on dark grey background
+// .su-brand-bar--white - Cardinal red wordmark on white background
+//
+// Markup: ../../templates/components/brand-bar/brand-bar.twig
+//
+// Style guide: Components.Brand Bar
+//
+
+.su-brand-bar {
+  @include centered-grid-container;
+  width: 100%;
+  height: 30px;
```

Comment:
Should the height of the brand bar be hardcoded in px? The wordmark is text, rendered in the ligature font. Should the height of the container adjust if the user changes font size? Or is our brand bar fixed height? If the latter, should the font-size be specified? Right now it's inheriting from `<body>`.

Reply:
I hardcoded it in px since it seems like it will always be 30px tall on any devices - thought that might be less room for surprises. I should definitely specify a font size....I guess that isn't dependent on anything (breakpoint eg) so I'll just hardcode it in. Thanks!

Reply:
Just added font size and pushed it up.

Reply:
Should the `font-size` go in the `logo` mixin with all the other font specifications?

Reply:
I see the `logo` mixin is also in the global footer, so you probably did put it in the right place. Sorry for my confusion

Reply:
I think I prefer it outside of the logo mixin since the font size of the ligature font is going to change depending on where it is used (eg, in lockup or footer it's going to be different).